servicecontrol/plugins/heartbeat.md

```diff
@@ -75,11 +75,18 @@ partial: ttlCode
 partial: disable
 
 
-## Expired heartbeat messages forwarded to Dead letter queue
+## Expired heartbeat messages
 
-Heartbeat messages have a time to be received (TTBR) set based on the TTL value. If ServiceControl does not consume the heartbeat messages before the TTBR expires then transports that support a dead letter queue (DLQ) will move these messages to the DLQ. This can happen when ServiceControl is stopped or very busy. The dead letter queue needs to be monitored and cleaned up.
+Heartbeat messages have a time to be received (TTBR) set based on the TTL value.  If ServiceControl does not consume the heartbeat messages before the TTBR expires then those messages may be discarded. Transports like MSMQ and ASB support DLQ and if configured the messages will be forwarded to the DLQ instead. 
 
 
 ### MSMQ
 
-If this dead letter queue behavior is not required then it can disabled. This frequently happens with MSMQ as NServiceBus configures the usage of the DLQ by default. Read [MSMQ connection strings](/nservicebus/msmq/connection-strings.md) on how to disabled dead letter queue usage.
+Although NServiceBus configures the use of DLQ by default, messages that are defined with TTBR will not be automatically forwarded to the DLQ and will be discared. 
```

Comment:
This could very well be my ignorance showing, but isn't this in contradiction with the previous statement?

>  Transports like MSMQ and ASB support DLQ and if configured the messages will be forwarded to the DLQ instead.

I read that as (with MSMQ) the message will be forward to DLQ if TTBR is after processing time.

Then we say:
>  messages that are defined with TTBR will not be automatically forwarded to the DLQ and will be discared

Which seems to suggest the opposite.